spec/features/idv/usps_verification_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'USPS verification' do
+  include SamlAuthHelper
+  include IdvHelper
+
+  context 'signing in when profile is pending USPS verification' do
```

Comment:
Also, I added a new scenario for the context where the user is signing back in directly (i.e. not coming from an SP). PTAL.